src/Plugin/Field/FieldFormatter/FieldFormatterBase.php

```diff
@@ -15,7 +15,7 @@
   /**
    * Entity view display.
    *
-   * @var \Drupal\Core\Entity\Display\EntityViewDisplayInterface
+   * @var \Drupal\Core\Entity\Display\EntityViewDisplayInterface[]
```

Comment:
Don't worry

On 16 Jul 2016 12:36 p.m., "Andy Postnikov" <notifications@github.com>
wrote:

> In src/Plugin/Field/FieldFormatter/FieldFormatterBase.php
> <https://github.com/drupal-media/field_formatter/pull/5#discussion_r71064413>
> :
>
> > @@ -15,7 +15,7 @@
> >    /**
> >     * Entity view display.
> >     *
> > -   * @var \Drupal\Core\Entity\Display\EntityViewDisplayInterface
> > +   * @var \Drupal\Core\Entity\Display\EntityViewDisplayInterface[]
>
> Hm, then I got confused by varname
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/drupal-media/field_formatter/pull/5/files/ada2ec3a0bfcdc06ba9dbbccba586a64e557bfa2#r71064413>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABz7nw-qFM3ryUi98acWfMlJoiJzt2Rks5qWLQxgaJpZM4JIyGi>
> .
>